README.md

```diff
@@ -1,2 +1,95 @@
-# navigation
-Repository for the discussion and research in to navigating from page to page whilst staying in immersive mode.
+# UA initiated immersive sessions
+
+## Motivation
+
+Current workflow to [request a WebXR session](https://www.w3.org/TR/webxr/#dom-xr-requestsession) requires user gesture activation. 2D Pages usually provide a button that users click or tap to trigger immersive mode.
+
+Some scenarios like in-vr navigation require entering immersive mode automatically without user gesture activation. This necessitates a standard mechanism for UAs to grant immersive sessions.
+
+## Design Goals
+
+The goal of this repository is to discuss and design a simple API for UAs to grant immersive sessions without an explicit user activation in specific scenarios.
```

Comment:
again, need to rephrase things to not focus on _activation_ ... it's more about explicit permission requests from the UA.